Content.Client/GameTicking/Managers/TitleWindowManager.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Content.Shared.CCVar;
+using Robust.Client;
+using Robust.Client.Graphics;
+using Robust.Shared;
+using Robust.Shared.Configuration;
+
+namespace Content.Client.GameTicking.Managers;
+
+public sealed class TitleWindowManager
+{
+    [Dependency] private readonly IBaseClient _client = default!;
+    [Dependency] private readonly IClyde _clyde = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    [Dependency] private readonly IGameController _gameController = default!;
+
+    public void Initialize()
+    {
+        _cfg.OnValueChanged(CVars.GameHostName, OnHostnameChange, true);
+        _cfg.OnValueChanged(CCVars.GameHostnameInTitlebar, OnHostnameTitleChange, true);
+
+        _client.RunLevelChanged += OnRunLevelChangedChange;
+    }
+
+    public void Shutdown()
+    {
+        _cfg.UnsubValueChanged(CVars.GameHostName, OnHostnameChange);
+        _cfg.UnsubValueChanged(CCVars.GameHostnameInTitlebar, OnHostnameTitleChange);
+    }
+
```

Comment:
Initialize is only called once upon game open (as it runs out of sim) so this doesn't really matter but it's also not hurting anyone so.

Reply:
I dunno I was told by nik to add it to not suffer someone's wrath

Reply:
It doesn't hurt anything to have it so it's fine.